src/parcsr_ls/par_ilu_setup.c

```diff
@@ -1773,7 +1776,9 @@ hypre_ILUSetupLDUtoCusparse(hypre_ParCSRMatrix  *L,
    hypre_CSRMatrixData(LDU_diag) = LDU_diag_data;
 
    /* now sort */
+#if defined(HYPRE_USING_GPU)
    hypre_CSRMatrixSortRow(LDU_diag);
```

Comment:
I am a little surprised that we didn't have this #if guard. @oseikuffuor1 , we don't need sorted LDU on CPU, right?

Reply:
Sorry I'm just seeing this. No we don't need it on CPU. Thanks Wayne.